vm/adb/adb.go

```diff
@@ -338,6 +338,14 @@ func (inst *instance) repair() error {
 	// Switch to root for userdebug builds.
 	inst.adb("root")
 	inst.waitForSSH()
+
+	// ADB connects to a phone and starts syz-fuzzer while the phone is still booting.
+	// 60 seconds delay should allow the phone to finish a boot process and start
+	// all services and processes. Otherwise syzkaller may create a race condition
```

Comment:
How exactly does this look? Say, if somebody will try to reduce this timeout in future, what do they need to look for? Or if this timeout is not enough for their device, how can they understand that they are seeing the same issue?